chips/lowrisc/src/hmac.rs

```diff
@@ -431,34 +431,3 @@ impl hil::digest::HmacSha512 for Hmac<'_> {
         Err(ErrorCode::NOSUPPORT)
     }
 }
-
-impl hil::digest::Sha256 for Hmac<'_> {
-    fn set_mode_sha256(&self) -> Result<(), ErrorCode> {
-        if self.busy.get() {
-            return Err(ErrorCode::BUSY);
-        }
-        let regs = self.registers;
-
-        // Ensure the SHA is setup
-        regs.cfg.write(
-            CFG::HMAC_EN::CLEAR
-                + CFG::SHA_EN::SET
-                + CFG::ENDIAN_SWAP::CLEAR
-                + CFG::DIGEST_SWAP::SET,
-        );
-
-        Ok(())
```

Comment:
Where does this happen instead?

Reply:
We still need to write the `cfg` though

Reply:
When would this get called today for an HMAC operation? Nothing would have a reference to the sha256 trait.